libs/features/mas/web-components/src/variants/ccd-suggested.js

```diff
@@ -0,0 +1,105 @@
+import { html, css } from 'lit';
+import { VariantLayout } from './variant-layout';
+import { CSS } from './ccd-suggested.css.js';
+
+const AEM_FRAGMENT_MAPPING = {
+  name: { tag: 'h4', slot: 'detail-m' },
+  title: { tag: 'h3', slot: 'heading-xs' },
+  price: { tag: 'p', slot: 'price' },
+  description: { tag: 'div', slot: 'body-xs' },
+  cta: { slot: 'cta', size: 's' },
```

Comment:
should be ` ctas: { slot: 'cta', size: 's' },`

Reply:
fixed